### DISCUSSION TITLE
Since 3.12.1, bundling fails

### DISCUSSION THREAD

**Discussion by zoispag** [alpinejs/alpine#3610] [7. Bugs (archived)]
Hi there,

Since 3.12.1, the PR created by Dependabot fails.
We are using the [Remove Flow Types](https://laravel-mix.com/extensions/remove-flow-types) and it fails because of 6d234578818bf031144f728ace3fb971541a5216

![image](https://github.com/alpinejs/alpine/assets/21138205/6aabf404-8bbf-46df-8ab3-85612e5b0ff9)

Is it possible to rewrite `return event.detail ?? event.target.value` in a "require-compatible" way?

Thanks.

**Comment by ekwoka** [comment]
You can have your bundle step before that transpile nullish coalescing, but this is an issue with `remove-flow-types` not with alpine.

Nullish coalescing is a JavaScript feature, so it's more on them to support it.

**Reply by zoispag** [reply]
I could not agree more, however, out of all JS packages that are bundled, this single 2 character `??` feature in Alpine codebase is breaking the Laravel mix compilation. Operationally, I find it definitely easier to "fix" it in Alpine, that having `remove-flow-types` package to support ESM modules (or what is really needed for this to work)

**Reply by ekwoka** [reply]
Are you gonna fix it in every application that uses nullish coalescing, a feature that is 3 years old?

That's impractical.

oh, I just checked, that package hasn't been updated in 4 years...

**Reply by zoispag** [reply]
Hm.. I just removed the `remove-flow-types` package completely.

I am getting a different but similar error on the same line:

![image](https://github.com/alpinejs/alpine/assets/21138205/ab055c29-a78f-47c1-b873-a22ad16d79de)


So it looks like it's not related to Flow but to Laravel Mix. Am I the only one that have this problem?

**Reply by ekwoka** [reply]
This definitely isn't the only package with nullish coalescing that someone might use with Laravel mix.

Looking through their issues or opening one there is the best course of action.

**Comment by SimoTod** [comment]
That package wraps a 7 years old dependency. It's probably a dead project, have you tried using https://babeljs.io/docs/babel-plugin-transform-flow-strip-types

**Reply by zoispag** [reply]
Thanks for the suggestion! I am not sure how to use it with Laravel Mix. I tried the `mix.babelConfig()` but it doesn't seem to work.
